### PR TITLE
Getting Started with AWS DynamoDB

### PR DESCRIPTION
**Pull request by Tsanguu**
added index.md and image files

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by Tsanguu** [comment]
This fixes #828

**Comment by hectorkambow** [comment]
👋 Thank you for submitting this PR - we will review it ASAP - all PRs go into our queue will be looked over before being approved to be published.

File: articles/getting-started-with-aws-dynamodb/index.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+
+#### Core Components of DynamoDB
+- **Tables:** Data is stored in tables. This is similar to other database systems. For instance, you can have a table that stores student details.
+- **Items:** A table is made up of zero or more items. Items can be compared to rows in a relational database. Each item is uniquely identifiable.
+- **Attributes:** Attributes are equivalent to columns or fields in a relational database. Each item has one or more attributes. For example, FirstName, LastName, etc. Items are allowed to have nested attributes.
+- **Primary key:** The primary key is unique for each item. DynamoDB has two types of primary keys:
+  - [Partition Key](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.CoreComponents.html#HowItWorks.CoreComponents.PrimaryKey): This forms a simple primary key with the partition key only. In this, the partition key is sufficient to identify each item uniquely.
```
Comment (kayere) [inline articles/getting-started-with-aws-dynamodb/index.md:26]: Please explain what a partition key is.

File: articles/getting-started-with-aws-dynamodb/index.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+  - [Partition key and sort key](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.CoreComponents.html#HowItWorks.CoreComponents.PrimaryKey): The two constitute a composite key. In instances where two items have a similar partition key, a sort key is used to identify them uniquely. Any two or more items that have a matching partition key should have a distinct sort key.
```
Comment (kayere) [inline articles/getting-started-with-aws-dynamodb/index.md:27]: and a sort key.

**Review by kayere — COMMENTED** [review]
Hello @Tsanguu, great article. Just make these few changes and the article will be good to go.

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by Tsanguu** [comment]
@kayere Well noted and rectified.

**Review by kayere — APPROVED** [review]

**Comment by Tsanguu** [comment]
This is for the peer-review process.

**Pull request merged**